### PR TITLE
chore: Alerts in design system.

### PR DESCRIPTION
**Pull request by markoa**
We're going to need this in a UI change coming soon.

Screenshot 2024-08-27 at 11 36 18

### CODE REVIEW

File: assets/js/components/Alerts/index.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import React from "react";
+
+import { IconAlertTriangleFilled, IconCircleXFilled, IconInfoCircleFilled } from "@tabler/icons-react";
+
+export function InfoAlert({ message, description }: { message: string | JSX.Element, description?: string | JSX.Element }) {
```
Comment (shiroyasha) [inline assets/js/components/Alerts/index.tsx:5]: The typing is identical for all three functions. Please extract with:

```
interface Props {
  message: string | JSX.Element;
  description: string | JSX.Element;
}
```

```
export function InfoAlert(props: Props) {
}
```


And then, extend it with a the TestableElement trait:

```
import { TestableElement } from "@/utils/testid";

interface Props extends TestableElement {
  message: string | JSX.Element;
  description: string | JSX.Element;
}
```

```
export function InfoAlert(props: Props) {
  return <div data-test-id={props.testId}>...
}
```

**Comment by shiroyasha** [comment]
The other thing I need to ask you is to test it out in dark mode, to make sure that all the elements we have in the design are working in both environments.

This was so far not possible on the design page, but I've just merged a PR that allows it https://github.com/operately/operately/pull/1005. You will need to rebase from main to test it.

**Comment by markoa** [comment]
@shiroyasha Thanks for the reminder about dark mode. Made it work:
Screenshot 2024-08-27 at 18 02 53
Screenshot 2024-08-27 at 18 02 59

I renamed them to callouts (inspired by Radix) because these widgets will not only be used as form alerts but also deeper on pages.

ℹ️  Two issues remain that I would tackle separately: #969 and secondary buttons not working well in dark mode.

**Comment by shiroyasha** [comment]
Looks good. Thanks for updating it for the darkness.

**Pull request merged**